packages/xo-server/src/xapi/index.mjs

```diff
@@ -1328,7 +1328,10 @@ export default class Xapi extends XapiBase {
         )
       ),
     ])
-    buffer = addMbr(buffer)
+    // only add the MBR for windows VM
+    if (vm.platform?.virdian === true) {
```

Comment:
Why `?.`? `vm.platform` should be defined for all VMs.

Also, you probably want to test with `'true'` the string, not the boolean.